api/v1beta1/policyautomation_types.go

```diff
@@ -52,6 +54,25 @@ type AutomationDef struct {
 	TowerSecret string `json:"secret"`
 	// JobTTL sets the time to live for the Kubernetes AnsibleJob object after the Ansible job run has finished.
 	JobTTL *int `json:"jobTtl,omitempty"`
+	// The maximum number of violation contexts will be provided to the Ansible Tower as extra variable.
```

Comment:
```suggestion
	// The maximum number of violating cluster contexts that will be provided to the Ansible job as extra variables.
```

Reply:
Can you document the default?

Reply:
Will do this after we decide on how to deal with the default case.